honeycombio/honeycomb-opentelemetry-node
# maint: Upgrade OpenTelemetry packages

## Which problem is this PR solving? 
Dependabot was having trouble upgrading some OTel packagaes because some of them needed up be upgraded together.

## Short description of the changes
- Upgrade OTel packages to latest for the Honeycomb SDK
- Upgrade OTel packages to latest for examples

## How to verify that this has the expected result
- Smoke tests pass
- Run the examples and verify that telemetry is being produced
